### ISSUE TITLE
Support for Python > 3.9?

### ISSUE THREAD

**Issue by milanmlft** [orthanc-server/orthanc-builder#19]
Hi Orthanc team,

Thanks for the great service! I was wondering whether there are any plans to support Python beyond 3.9 in the Docker images?
Or what the reason is to stick to this version?

Thank you!

**Comment by amazy** [comment]
Hi,

Actually, the python version is driven by the base image - currently Debian bullseye.
We should soon update to Debian bookworm and python will be upgraded to 3.11.

**Comment by milanmlft** [comment]
Great! Thanks for the reply! It might be useful to document this somewhere?

**Comment by amazy** [comment]
Good idea ! I will include this kind of block in the future Dockerhub readme:
```
System component                      version
---------------------------------------------
Base Debian image                     12 (bookworm)
Python                                3.11
Lua                                   5.4
```
